source/dotnet/wholesale-api/CalculationResults/CalculationResults.UnitTests/Infrastructure/CalculationResultClient/DatabricksSqlResponseParserTests.cs

```diff
@@ -35,7 +35,7 @@ public DatabricksSqlResponseParserTests()
 
     [Theory]
     [AutoMoqData]
-    public void Create_ReturnsResponseWithCorrectState(DatabricksSqlResponseParser sut)
+    public void Parse_ReturnsResponseWithCorrectState(DatabricksSqlResponseParser sut)
```

Comment:
General suggestion: How about the following to stick to the already used term "expected" inside all test implementations?
```suggestion
    public void Parse_ReturnsResponseWithExpectedState(DatabricksSqlResponseParser sut)
```